andela/codepirates-ah-frontend
# #167313356 setup react routing

#### What does this PR do?

- Setup react-router version LTS (React-router-dom)

#### Description of Task to be completed?

- as a user, I should be able to navigate through different pages using routes 

#### How should this be manually tested?

- by cloning this repo run the app locally and navigate to `/privacy` you will be able to see privacy page 

- navigation to any other routes wi=ch is not included it will bring 404 page 


#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#167313356](https://www.pivotaltracker.com/story/show/167313356)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/38810299/65634310-3cf60280-dfde-11e9-8f8a-5ac959208462.png)
![image](https://user-images.githubusercontent.com/38810299/65634411-70d12800-dfde-11e9-9501-df6e76fb6e9d.png)
![image](https://user-images.githubusercontent.com/38810299/65634426-77f83600-dfde-11e9-97a1-4b7d1e8b11f9.png)


#### Questions:
